doc/source/atomgroup.rst

```diff
@@ -231,42 +222,40 @@ These methods return a user-ordered :class:`~MDAnalysis.core.groups.AtomGroup` t
 Empty AtomGroups
 ----------------
 
-Empty AtomGroups can be constructed from each method. For example, using selection language::
+Empty AtomGroups can be constructed from each method. For example, using selection language:
```

Comment:
We don't have a dedicated construct so we should put down a canonical way to make an empty AG. Syntax-wise I liked best the one from the example further down
```python
null = u.atoms[[]]
```
which does
```python
null = mda.AtomGroup([], u)
```

In this way we can standardize by example and then you can show the other ways in which empty groups can occur.